velox/exec/WindowBuild.h

```diff
@@ -76,6 +76,10 @@ class WindowBuild {
     return data_->estimateRowSize();
   }
 
+  column_index_t inputChannelOf(column_index_t originalIndex) const {
```

Comment:
I don't think this function is needed after the changes in https://github.com/facebookincubator/velox/pull/9266